app/src/main/res/drawable/buttonbackground1.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="rectangle">
+            <corners android:radius="60dp"/>
+            <solid android:color="#189E52"/>
```

Comment:
Colours (in all the files it seems) need to use the variables as mentioned in the other PRs